core/datatrak.c

```diff
@@ -123,6 +123,23 @@ static int dtrak_prepare_data(int model, device_data_t *dev_data)
 	return g_models[i].libdc_num;
 }
 
+/*
+ * Return a default name for a tank based on it's size.
+ * Just get the first in the user's list for given size.
+ * Reaching the end of the list means there is no tank of this size.
+ */
+static char *cyl_type_by_size(int size)
```

Comment:
Sorry for being pedantic: But could we change the function signature to:
```
static const char *cyl_type_by_size(int size)
```
?
Returning `char *` softly implies that the caller owns the string and should free it. Returning `const char *` implies that this is owned by orhers and caller should duplicate it if it wants to take ownership. There are exceptions though.

Reply:
> Sorry for being pedantic

You are not, it's a good practice.
Have just sent another patch.